apps/AppLibs.mk

```diff
@@ -56,8 +56,7 @@ include $(TOPDIR)/ProtectedLibs.mk
 
 ifeq ($(CONFIG_BUILD_PROTECTED),y)
```

Comment:
This file is not a part of apps folder anymore. Let's move another location and rename.

Reply:
OK. I removed it and directly added 2 lines to the micom and wifi Makefile.